OracleAccessManagement/dockerfiles/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,447 @@
+Building an Oracle Access Management Image using Dockerfile Samples
+==========================================================================
+Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This Image includes binaries for Oracle Access Management (OAM) Release 12.2.1.4.0 and it has capability to create FMW Infrastructure domain and OAM specific Managed Servers.
+
+***Image***: oracle/oam:<version; example:12.2.1.4.0>
+
+## Prerequisites
+The following prerequisites are necessary before building OAM Docker images:
+
+* A working installation of Docker 18.03 or later
+
+## Prerequisites for OAM on Kubernetes
+
+Refer to the [Prerequisites](https://oracle.github.io/fmw-kubernetes/oam/prerequisites) in the Oracle Access Management Kubernetes documentation.
+
+
+## How to build
+This project offers a sample Dockerfile and scripts to build an Oracle Access Management 12cPS4 (12.2.1.4) image. 
+
+Building your own OAM image involves the following steps:
```

Comment:
Building the OAM image should require building the Oracle Server JRE and FMW Infrastructure images, not pulling them.